src/homework11/OutOfMemorySimulator.java

```diff
@@ -0,0 +1,7 @@
+package homework11;
+
+public class OutOfMemorySimulator {
+    public void emulateException() {
+        Float[] array = new Float[10000 * 100000];
```

Comment:
Does it really cause OutOfMemory?

Reply:
Yes, Java heap space error occurs.
![OutOfMemory](https://user-images.githubusercontent.com/58294194/72633844-00219300-3962-11ea-9b8c-f0eaf855d8a2.png)